examples/official/trial/mnist_gan_pytorch/model_def.py

```diff
@@ -1,10 +1,11 @@
 """
-This example shows how to interact with the new Determined PyTorch interface with multiple models,
-optimizers, and LR schedulers support to build a GAN network.
+This example demonstrates how to train a GAN with Determined's PyTorch API.
 
-In the new interface, you need to instantiate your models, optimizers, and LR schedulers in
-__init__ and run forward and backward passes and step the optimizer in train_batch. By doing so,
-you could be flexible in building your own training workflows.
+This example uses the new PyTorch API introduced in Determined 0.12.13, which
```

Comment:
question: why do we want to talk about a version number and when it is able to be removed?

Reply:
Yeah, this is a good point. I think it is similar to talking about the "new interface": at some point we want to remove "new" and just talk about the "PyTorch interface". I'd personally err on the side of doing that sooner rather than later -- maybe even now.